frameworks/helloworld/integration/tests/test_sidecar.py

```diff
@@ -0,0 +1,42 @@
+import dcos.http
+import json
+import pytest
+import re
+import shakedown
+
+from tests.test_utils import (
+    PACKAGE_NAME,
+    check_health,
+    get_marathon_config,
+    get_deployment_plan,
+    get_sidecar_plan,
+    get_task_count,
+    install,
+    marathon_api_url,
+    request,
+    run_dcos_cli_cmd,
+    uninstall,
+    spin,
+    start_sidecar_plan
+)
+
+
+def setup_module(module):
+    uninstall()
+    options = {
+        "service": {
+            "spec_file": "sidecar.yml"
+        }
+    }
+
+    install(None, PACKAGE_NAME, options)
+
+
+@pytest.mark.sanity
+def test_deploy():
+    get_deployment_plan()
```

Comment:
This doesn't succeed unless it returns a 200 which indicates the plan is COMPLETE.